tools/nuget-dump/nuget-dump.cs

```diff
@@ -0,0 +1,84 @@
+using System;
```

Comment:
Please choose better name e.g. nuget2ignored or nuget-hash-extractor or something like that